### ISSUE TITLE
'WorkflowBenchmark' object has no attribute 'tasks'

### ISSUE THREAD

**Issue by andersonandrei** [wfcommons/wfcommons#39]
**WfCommons Information**

- WfCommons version: 1.0
- Python Version: 3.11.2
- OS: Debian 12

**Describe the bug**

The `create_benchmark`function inside `WorkflowBenchmark` is not working
I'm running the following example from the tutorial (https://docs.wfcommons.org/en/latest/generating_workflow_benchmarks.html#) :

```
import pathlib

from wfcommons import BlastRecipe
from wfcommons.wfbench import WorkflowBenchmark

# create a workflow benchmark object to generate specifications based on a recipe
benchmark = WorkflowBenchmark(recipe=BlastRecipe, num_tasks=500)
# generate a specification based on performance characteristics
path = benchmark.create_benchmark(save_dir=pathlib.Path("/tmp/"), cpu_work=100, data=10, percent_cpu=0.6)
```
And getting the following error:

```
INFO:wfcommons.wfgen.generator:Generated a synthetic workflow with 498 tasks
Traceback (most recent call last):
  File "../WfCommons/tests/generate1.py", line 9, in <module>
    path = benchmark.create_benchmark(save_dir=pathlib.Path("/tmp/"), cpu_work=100, percent_cpu=0.6, data=10, gpu_work=0.6)
           
  File "../.local/lib/python3.11/site-packages/wfcommons/wfbench/bench.py", line 279, in create_benchmark
    for task in self.tasks.values():
                ^^^^^^^^^^
AttributeError: 'WorkflowBenchmark' object has no attribute 'tasks'
```

**Comments:**

Looking to the definition of a Workflow object back in https://github.com/wfcommons/WfCommons/blob/3825ff47931bfd0fc88ce773830350e3bd55e007/wfcommons/common/workflow.py , it seems that `Workflow` does not have 'tasks' as attribute, but 'WorkflowBenchmark', back in "wfcommons/wfbench/bench.py", line 279, in create_benchmark, seems to be trying to loop over 'tasks'.

**Additional context**

Debugging other example (https://docs.wfcommons.org/en/latest/generating_workflow_benchmarks.html#generate-from-synthetic-workflow-instances) I figured out that the 'data' argument is not used anymore at `create_benchmark_from_synthetic_workflow` and `create_benchmark`. So I also tried to run the code above removing the `data` argument from the final line, but the problem with `tasks` remains the same:

```
import pathlib

from wfcommons import BlastRecipe
from wfcommons.wfbench import WorkflowBenchmark

# create a workflow benchmark object to generate specifications based on a recipe
benchmark = WorkflowBenchmark(recipe=BlastRecipe, num_tasks=500)
# generate a specification based on performance characteristics
path = benchmark.create_benchmark(save_dir=pathlib.Path("/tmp/"), cpu_work=100, percent_cpu=0.6)
```

```
INFO:wfcommons.wfgen.generator:Generated a synthetic workflow with 498 tasks
Traceback (most recent call last):
  File "../WfCommons/tests/generate1.py", line 9, in <module>
    path = benchmark.create_benchmark(save_dir=pathlib.Path("/tmp/"), cpu_work=100, percent_cpu=0.6, data=10, gpu_work=0.6)
           
  File "../.local/lib/python3.11/site-packages/wfcommons/wfbench/bench.py", line 279, in create_benchmark
    for task in self.tasks.values():
                ^^^^^^^^^^
AttributeError: 'WorkflowBenchmark' object has no attribute 'tasks'
```

Can you help me with that, please? 

Thank you in advance!

**Comment by rafaelfsilva** [comment]
Hi @andersonandrei , it looks like the issues above have been fixed by the PRs you submitted (#40, #41). Could you confirm? Thank you

**Comment by andersonandrei** [comment]
Hi @rafaelfsilva, yes, that is right. Thank you!
